live-build/misc/migration-scripts/dx_verify

```diff
@@ -37,16 +38,29 @@ function usage() {
 
 function cleanup() {
 	if ! $DEBUG; then
-		if [ -n "$root" ]; then
+		if [[ -n "$root" ]]; then
```

Comment:
Same question here.

Reply:
Yeah, if we can't parse the root dataset name properly: `[[ -n $rds ]] || die "could not find migration dataset for version $version"`